publication-fanout-handler/src/main/java/no/unit/nva/publication/events/DynamodbStreamRecordPublicationMapper.java

```diff
@@ -88,7 +88,7 @@ private static Item toItem(Map<String, com.amazonaws.services.dynamodbv2.model.A
         return result;
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings({"unchecked","PMD.CognitiveComplexity"})
```

Comment:
This method is complex but it is a copy paste from aws code and  hopefully one day will be removed